htdocs/web_portal/views/site/site_downtimes.php

```diff
@@ -4,20 +4,28 @@
 ?>
 <div class="rightPageContainer">
     <div style="overflow: hidden">
+        <div style="float: left;">
+            <img src="<?php echo \GocContextPath::getPath()?>img/down_arrow.png" class="pageLogo" />
+        </div>
         <div style="float: left;">
             <h1 style="float: left; margin-left: 0em; padding-bottom: 0.3em;">
-                Downtimes Affecting
                 <a  style="font-family: inherit; font-size: inherit; font-weight: inherit; padding-bottom: inherit; "
                     href="index.php?Page_Type=Site&id=<?php echo $site->getId(); ?>">
-            <?php xecho($site) ?></a>'s
-                Services
+                <?php xecho($site) ?></a> Downtimes
             </h1>
         </div>
+        <div style="float: left;">
+            <span style="clear: both; float: left; padding-bottom: 0.4em;">
+                All downtimes affecting <?php xecho($site) ?> services and endpoints (Year-Month-Day Time in UTC)
```

Comment:
You've moved the "(Year-Month-Day Time in UTC)" bit to a subtitle? Feels more appropriate in the listContainer.

Reply:
Changed to look like this -
![image](https://user-images.githubusercontent.com/20706670/66848965-f278fd80-ef6d-11e9-92d4-440d6d278016.png)